frontend/src/pages/Book.jsx

```diff
@@ -21,17 +22,32 @@ export default function Book() {
 	}, []);
 
 	return (
-		<>
-			<h2 className="PageTitle">{book.title}</h2>
-			<dl>
-				<dt>author</dt>
-				<dd>{book.author}</dd>
-				<dt>genre</dt>
-				<dd>{book.genre}</dd>
-				<dt>year</dt>
-				<dd>{book.yearPublished}</dd>
-			</dl>
-			<img src={`https://picsum.photos/200/300`} alt="book cover" width={200} height={300} />
+		<> 
```

Comment:
@Shinh18 You are missing a heading on the page, don't know where we sould put it...
the class names are not really coherent with the rest of the app you should look into the BEM methodology
for exemple : https://webuild.envato.com/blog/how-to-scale-and-maintain-legacy-css-with-sass-and-smacss/
you can also look at how I named in other components.